tests/5.0/loop/test_loop_bind.c

```diff
@@ -0,0 +1,151 @@
+//===--- test_loop_bind.c ---------------------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// This test checks the loop directive with the order(concurrent) clause. The
+// order(concurrent) clause is assumed to be present if it is not present, so
+// this test covers the standalone loop directive as well. The test creates a
+// parallel region with a loop construct nested within, and performs simple
+// operations on an int array which are then checked for correctness. Note
+// that the number of
+//
+////===----------------------------------------------------------------------===//
+#include <assert.h>
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 32
+
+int test_loop_bind_teams() {
+  OMPVV_INFOMSG("test_loop_bind_teams");
+  int errors = 0;
+  int x[N];
+  int y[N];
+  int z[N];
+  int num_teams = -1;
+
+  for (int i = 0; i < N; i++) {
+    x[i] = 1;
+    y[i] = i;
+    z[i] = 2*i;
+  }
+
+#pragma omp teams num_teams(OMPVV_NUM_TEAMS_DEVICE) thread_limit(OMPVV_NUM_THREADS_HOST)
+  {
+#pragma omp loop bind(teams)
+    for (int i = 0; i < N; i++) {
+      x[i] += y[i]*z[i];
```

Comment:
I expect all master threads in the number of teams will access the same x. Adding teams binding inside teams is redundant, but we could try both combinations.